### PR TITLE
Malformed XML comment

### PR DESCRIPTION
**Pull request by mihailik**
Fails the build when XML doc generation is enabled.

### CODE REVIEW

File: src/Common/src/System/Net/Http/HttpHandlerDefaults.cs [hunk 1]
```diff
@@ -4,7 +4,7 @@
 namespace System.Net.Http
 {
     /// <summary>
-    /// Defines default values for http handler properties which is meant to be re-used across WinHttp & UnixHttp Handlers
+    /// Defines default values for http handler properties which is meant to be re-used across WinHttp &amp; UnixHttp Handlers
```
Comment (stephentoub) [inline src/Common/src/System/Net/Http/HttpHandlerDefaults.cs]: Can we just change it to the word "and"?

**Comment by mihailik** [comment]
Nice idea, @stephentoub -- and done.

**Comment by stephentoub** [comment]
Thanks. LGTM. Can you squash this?

**Comment by stephentoub** [comment]
Replaced by #4185.  I just cherry-picked and squashed these commits; authorship remains yours.  Thanks for the fix.

**Pull request closed without merging**